tests/test_gslide.py

```
@@ -0,0 +1,81 @@
+        
+        # 4. Add text to shape
+        shape_text = "Created with Service Account Authentication!"
+        text_result = service.add_text_to_slide(
+            presentation_id, 
+            slide_id, 
+            shape_id, 
+            shape_text,
+            {'bold': True, 'foregroundColor': {'opaqueColor': {'rgbColor': {'red': 0.1, 'green': 0.5, 'blue': 0.8}}}}
+        )
+        self.assertIsNotNone(text_result)
+        print(f"Added text to shape")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)